components/avatar/avatar.vue

```diff
@@ -12,7 +12,7 @@
       <!-- @slot Slot for avatar content -->
       <slot v-if="showDefaultSlot" />
       <span
-        v-else-if="showInitials"
+        class="d-ps-absolute d-zi-base"
```

Comment:
Sure, I was thinking the same, I just wanted to have the image over the initials because the initials are always being displayed so when the image is loading, the initials are behind.

 But I'll need to test several cases to check that z-index would work correctly in all of them.

Reply:
Might work if you just move the slot after the initials in the DOM order.